README.md

```diff
@@ -18,6 +18,33 @@ for issues to get started.
 
 For more technical details, see [our wiki](http://wiki.languagetool.org).
 
+#### Scripted installation and building
+To install or build using a script, simply type:
+```
+curl -L https://bit.ly/2kfvAha | sudo bash <options>
```

Comment:
If the script is included, there is no reason to retrieve the on-line version. Besides, it would have to be a static version to be possible to be reviewed.

Reply:
So they should first do git clone, then execute the install script? If you are doing git over https, I'm not sure how much of a difference security-wise that is.